packages/core/core/src/Parcel.js

```diff
@@ -100,6 +100,13 @@ export default class Parcel {
 
     await initSourcemaps;
     await initRust?.();
+    try {
+      await initSentry?.();
+    } catch (e) {
+      // Fallthrough
+      // eslint-disable-next-line no-console
+      console.warn(e);
```

Comment:
It's fine to continue startup if sentry fails, but we should log a bit more effectively here. `logger` is available and we should provide some context to the error